src/platforms/react-native/sourcemaps.mdx

```diff
@@ -0,0 +1,102 @@
+
+On iOS you will need to [revert the build script "Bundle React Native code and images"](/platforms/react-native/manual-setup/manual-setup/#bundle-react-native-code-and-images).
+
+On Android you will need to [remove the gradle integration](/platforms/react-native/manual-setup/manual-setup/#enable-gradle-integration).
+
+## Preparing the Source Maps for a Release
+
+You will need to generate and upload the source maps at build time for every release of your app for the events sent to be correctly unminified.
+
+### Configure CLI
+
+Configure the CLI by reviewing the [configuration guide](https://docs.sentry.io/product/cli/configuration/) for CLI commands.
+
+### Generating the Bundle and Source Maps
+
+You would use the React Native CLI to generate the JavaScript bundle and source maps for your app:
+
+```bash {tabTitle:Android}
+react-native bundle \
+  --dev false \
+  --platform android \
+  --entry-file index.android.js \
+  --bundle-output index.android.bundle \
+  --sourcemap-output index.android.bundle.map
+```
+
+```bash {tabTitle:iOS}
+react-native bundle \
+  --dev false \
+  --platform ios \
+  --entry-file index.ios.js \
+  --bundle-output main.jsbundle \
+  --sourcemap-output main.jsbundle.map
+```
+
+[You can read more about the React Native CLI Bundle command here.](https://github.com/react-native-community/cli/blob/master/docs/commands.md#bundle)
+
+#### Input
+
+You would pass your entry file to `--entry-file` for Android or iOS respectively. By default this file is usually named `index.js`. It is often `index.android.js` and `index.ios.js` for Android and iOS respectively.
+
+#### Outputs
+
+This will output your bundle file and source maps. By default these files are called
+
+- iOS: `main.jsbundle` and `main.jsbundle.map`
+- Android: `index.android.bundle` and `index.android.bundle.map`
+
+<Note>
+Make sure that the sourcemap's name is the bundle's name + ".map" for source maps to correctly be detected.
+</Note>
+
+### Uploading the Bundle and Source Maps
+
+```bash {tabTitle:Android}
+node_modules/@sentry/cli/bin/sentry-cli releases \
+    files <release> \
+    upload-source maps \
+    --dist <dist> \
+    --strip-prefix /path/to/project/root \
+    --rewrite index.android.bundle index.android.bundle.map
+```
+
+```bash {tabTitle:iOS}
+node_modules/@sentry/cli/bin/sentry-cli releases \
+    files <release> \
+    upload-source maps \
+    --dist <dist> \
+    --strip-prefix /path/to/project/root \
+    --rewrite main.jsbundle main.jsbundle.map
+```
+
+<Note>
+
+If you're using `sentry-cli` prior to version 1.59.0, pass `--rewrite` to the `upload-source maps` command to fix up the source maps before the upload (inlines sources and so forth). Version 1.59.0 does this automatically.
+
+</Note>
+
+## Troubleshooting
+
+If source maps are still not recognized, check for warnings similar to:
+
+![Missing sources warning](001-manual-source maps.png)
```

Comment:
@PeloWriter Hmm this image doesn't seem to be showing on the [preview](https://sentry-docs-aqxym6aw2.sentry.dev/platforms/react-native/sourcemaps/#troubleshooting)?

Reply:
I don't see it in the list of files in the PR, so perhaps it didn't get added? It needs to be in the same directory. Also note, that it *may* be missing a hyphen between "source" and "maps"?

Reply:
@PeloWriter That extra space was it, but also the fact that I didn't move the file from the `manual-setup` folder, thanks!